.github/workflows/deploy_docs.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Deploy Schema Docs'
+
+on:
+  push:
+    branches:
+      - "main"
+
+jobs:
+  deploy-docs:
+    if: github.actor != dependabot[bot]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger Deploy workflow
+        uses: CircleCI-Public/trigger-circleci-pipeline-action@v1.0.5
+        with:
+          target-slug: github/labforward/laboperator-schema-docs
+        env:
+          CCI_TOKEN: ${{ secrets.CCI_TOKEN }}
```

Comment:
since it's a public repo, it doesn't have access to the organization `CCI_TOKEN`.. and I couldn't find a way to grant access to the token in a secure way in GitHub.. should we set this up using circleci instead of github action? you can then use the `circleci-api-token`, which are already configured to be accessible only to organisation member..

Reply:
@kgrhartlage what do you think? Should we just go with the path of least resistance? 

I can try to set it up using circleCI to trigger the workflow; However AFAIK there is currently no (direct) way of triggering an entire CCI workflow through the API, you have to use some workarounds.

Reply:
you'll need to [trigger a new pipeline](https://circleci.com/docs/api/v2/index.html#operation/triggerPipeline), instead of triggering specific workflow directly..

you can provide parameters with the pipeline trigger, and then use it to define the condition for the expected workflow to run..